packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -59,7 +102,48 @@ class FlutterErrorDetails {
 
   /// A human-readable description of where the error was caught (as opposed to
   /// where it was thrown).
-  final String context;
+  String get context {
+    if (_contextName == null && _contextObject == null)
+      return null;
+    return diagnosticContext('')?.toStringDeep();
+  }
+
+  // XXX make this private
```

Comment:
Can we make this private?

Alternatively, change to a TODO?

Reply:
yes! done.